src/game-logic/reducers/applyCrows.js

```diff
@@ -0,0 +1,45 @@
+import { itemType } from '../../enums'
+import { itemsMap } from '../../data/maps'
+import { getPlotContentType } from '../../utils'
+import { CROW_CHANCE } from '../../constants'
+import { CROW_ATTACKED } from '../../templates'
+
+import { fieldHasScarecrow, updateField } from './helpers'
+
+/**
+ * @param {farmhand.state} state
+ * @returns {farmhand.state}
+ */
+export const applyCrows = state => {
```

Comment:
just noticing there seems to be no spec for this one. wanted to flag it in case it got missed, but if none existed it doesn't have to block this PR

Reply:
Added a TODO for this in https://github.com/jeremyckahn/farmhand/commit/b28490644c45a589008d909fc85add26f51d46df.